apps/namadillo/src/hooks/useValidatorFilter.tsx

```diff
@@ -13,7 +13,9 @@ const filterValidators = (validator: Validator, search: string): boolean => {
   const preparedSearch = search.toLowerCase().trim();
   return (
     validator.address.toLowerCase().indexOf(preparedSearch) > -1 ||
-    validator.alias.toLowerCase().indexOf(preparedSearch) > -1
+    (validator.alias || validator.address)
```

Comment:
Minor nitpick, but I think this will search the address twice if the alias is undefined, so you could probably just use `validator.alias?` or something like that.